lib/collection/response.js

```diff
@@ -496,13 +738,14 @@ _.assign(Response, /** @lends Response */ {
     },
 
     /**
-     * @private
+     * @public
+     * Guesses the mime type by lookup in mimeFormat
+     * parses the filename from content disposition header
```

Comment:
Extracts the mime related information from the response headers. Looks up mime type and extracts `filename` from content disposition header.